tap_adroll/client.py

```diff
@@ -41,6 +27,39 @@ def __init__(self, config_path, config):
             LOGGER.info("Error initializing AdrollClient during token refresh, please reauthenticate.")
             raise AdrollAuthenticationError(e)
 
+    def authenticate_request(self):
+        token = {
+            'access_token': self.config['access_token'],
+            'refresh_token': self.config['refresh_token'],
+            'token_type': 'Bearer',
+            # Set expires_in to a negative number to force the client to reauthenticate
+            'expires_in': '-30'
+        }
+        extra = {
+            'client_id': self.config['client_id'],
+            'client_secret': self.config['client_secret']
+        }
+        if self.dev_mode :
+            self.access_token = self.config.get('access_token')
```

Comment:
We do not use the access_token outside of this function, don't make this a field on the class.